server-spi-private/src/main/java/org/keycloak/models/ClientPolicyConstants.java

```diff
@@ -0,0 +1,5 @@
+package org.keycloak.models;
+
+public class ClientPolicyConstants {
+    public static final String USE_LIGHTWEIGHT_ACCESS_TOKEN_ENABLED = "client.use.lightweight.access.token.enabled";
```

Comment:
Nitpick: Since we have single constant, maybe it is better to re-use existing class `Constants` in the same package? And possibly just add some section to that class like:
```
// CLIENT POLICY CONSTANTS

// KeycloakSession attribute to specify if to use lightweight access token when issuing tokens for current client
public static final String USE_LIGHTWEIGHT_ACCESS_TOKEN_ENABLED = "client.use.lightweight.access.token.enabled";

```

Reply:
Thank you for your comment.
Fixed to use the `Constants` class.

Reply:
Thanks